[2024-05-10T06:25:04.439+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-10T06:25:04.573+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Proyecto_SpaceX_v1.Respuesta_Satelite scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2024-05-10T06:25:04.593+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Proyecto_SpaceX_v1.Respuesta_Satelite scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2024-05-10T06:25:04.596+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-10T06:25:04.612+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Respuesta_Satelite> on 2024-03-27 00:00:00+00:00
[2024-05-10T06:25:04.620+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2198) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-10T06:25:04.622+0000] {standard_task_runner.py:63} INFO - Started process 2200 to run task
[2024-05-10T06:25:04.622+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Proyecto_SpaceX_v1', 'Respuesta_Satelite', 'scheduled__2024-03-27T00:00:00+00:00', '--job-id', '1312', '--raw', '--subdir', 'DAGS_FOLDER/Proyecto_SpaceX_v1.py', '--cfg-path', '/tmp/tmpwgxlsmbv']
[2024-05-10T06:25:04.624+0000] {standard_task_runner.py:91} INFO - Job 1312: Subtask Respuesta_Satelite
[2024-05-10T06:25:04.670+0000] {task_command.py:426} INFO - Running <TaskInstance: Proyecto_SpaceX_v1.Respuesta_Satelite scheduled__2024-03-27T00:00:00+00:00 [running]> on host 1b7f0b10dd99
[2024-05-10T06:25:04.744+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Proyecto_SpaceX_v1' AIRFLOW_CTX_TASK_ID='Respuesta_Satelite' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T00:00:00+00:00'
[2024-05-10T06:25:04.793+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-10T06:25:04.819+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-10T06:25:04.822+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-10T06:25:04.842+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Proyecto_SpaceX_v1, task_id=Respuesta_Satelite, run_id=scheduled__2024-03-27T00:00:00+00:00, execution_date=20240327T000000, start_date=20240510T062504, end_date=20240510T062504
[2024-05-10T06:25:04.878+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-10T06:25:04.918+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-10T06:25:04.928+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
